src/transports/winhttp.c

```diff
@@ -786,6 +790,14 @@ static int winhttp_connect(
 		goto on_error;
 	}
 
+	if (!WinHttpSetOption(t->session,
```

Comment:
What happens if we are on the unpatched win7? Will this call return an error because the os does not know about TLS1_2? Should we perhaps ignore the return code then?